README.md

```diff
@@ -1 +1,49 @@
-# Portfolio-Website
\ No newline at end of file
+# Portfolio-Website
+Portfolio: setup and mobile version skeleton
+
+## Built With
+
+- HTML
+- CSS
+- Technologies used
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```

Comment:
- [ ] Kindly specify the prerequisites for someone to be able to run this project on their computer if they clone it. This is because not everyone can run the app instantly when cloned. There are some technologies one must install in their computer and browser to have to run the app, etc...
- Also how to set up the app locally on ones' computer
- As well as how to use the app. For tests, that will come up in later chapters in the course program. Please work on them and remove the one you don't have a definition for. 👍

Reply:
But I really don't know. please let me know how to do this.

Reply:
- [ ] You have not yet implemented the changes requested by the previous reviewer, kindly check on the suggestions left by the reviewer, you can check this [readme file](https://github.com/danmainah/music-instruments-store/blob/main/README.md) for inspiration.